consensus/mempool_test.go

```diff
@@ -261,3 +261,8 @@ func (app *CounterApplication) PrepareProposal(
 	req abci.RequestPrepareProposal) abci.ResponsePrepareProposal {
```

Comment:
There are two files that contain changes in the original PR:
* (internal)/consensus/common_test.go These changes change `makeState` but it's not clear to me how important that is (it seems a convenience change). @williambanfield, what to do think?
* (internal)/consensus/state_test.go In this file, test case `TestProcessProposalAccept` is written from scratch. It seems relevant for the code of this PR. Please port it to this PR

Reply:
:+1: to the second change. As per the first change, that actually is necessary to make the first change doable, so both should be done.

Reply:
Lol I didn't realise how much clean up work was done in these tests between v0.34 and v0.35

Reply:
This is ridiculous